src/aten/sycl/BinaryKernels.cpp

```diff
@@ -59,11 +95,11 @@ void add_kernel(TensorIterator& iter, const c10::Scalar& alpha) {
   }
 }
 
-void sub_kernel(TensorIterator& iter, const c10::Scalar& alpha) {
+void sub_kernel(TensorIteratorBase& iter, const c10::Scalar& alpha) {
```

Comment:
Why using TIbase? not TI? CUDA is using it?

Reply:
Align to cuda's impl